compiler/rustc_error_messages/src/lib.rs

```diff
@@ -43,6 +43,7 @@ fluent_messages! {
     passes => "../locales/en-US/passes.ftl",
     privacy => "../locales/en-US/privacy.ftl",
     typeck => "../locales/en-US/typeck.ftl",
+    rustc_mir_dataflow => "../locales/en-US/rustc_mir_dataflow.ftl",
```

Comment:
```suggestion
    mir_dataflow => "../locales/en-US/rustc_mir_dataflow.ftl",
```

We can just call this `mir_dataflow` everywhere.